contribute/contribution-guidelines/upgrade-module.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Upgrade Contributions
+weight: 60
+---
+
+# Upgrade Contributions: Best Practices and Essential Tips
+
+There are elements to consider if a modification has been made to the structure of the database in a contribution to the PrestaShop project, specifically a contribution to the Core.
+
+During a contribution, if there's a change in the structure of the given database, the "Needs autoupgrade PR" label is added. It indicates that another pull request is required to ensure the proper functioning of the upgrade process to the relevant version.
+
+![Screenshot of the Label](../../img/needs-autoupgrade-pr-label.png)
+
+This label is the result of verifications executed on each pull request, where a comparison of stores is done against the base and your branch. If differences are found, this means a store upgrading to a version containing your changes will require its content to be migrated.
+
+It is essential to keep the upgrade process up to date according to the changes made to the Core to ensure the proper functioning of upgrades for different versions of PrestaShop.
+
+## How to determine which content is affected and what needs to be added ?
+
+When there's a modification to the `db-structure.sql` file or if a Doctrine entity has been modified, added, or deleted, it impacts the database structure. Modifying the list of hooks and feature flags has consequences on the upgrade process as well.
+
+These changes can affect how the application interacts with the database and how upgrade procedures are handled.
+
+It's important to consider these aspects when contributing to ensure a smooth upgrade experience for users.
+
+{{% notice tip %}}
+The Core command prestashop:schema:update-without-foreign allows you to update the database according to modifications made to the Doctrine entities during the development phase. It outputs the queries executed for each change detected.
+Please note, after restarting the command the requests will have already been executed and the output will be empty.
```

Comment:
On v9 there are no two options `--dump-sql` and `--force` I don't know if it's worth describing them here. But fi you don't use the `--force` option what you claim here is wrong, the DB will not be modifed

Reply:
Thank you for the clarification, the doc goes back before the creation of PR removing the execution of the doctrine command from the core

Reply:
Yeah and the PR is not merged yet anyway https://github.com/PrestaShop/PrestaShop/pull/36233
But I anticipate the future new behaviour